libs/CMakeLists.txt

```diff
@@ -27,3 +27,4 @@ if (USE_INTERNAL_CONSISTENT_HASHING_LIBRARY)
     add_subdirectory (consistent-hashing)
 endif ()
 add_subdirectory (consistent-hashing-sumbur)
+add_subdirectory (libwidechar_width)
```

Comment:
Missing LICENCE and README in the library directory.
It must mention exact source (URL, commit) of library.

... but you can use submodule to provide all of these automatically.